src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/ValidateApplicantServiceController.java

```diff
@@ -44,18 +45,19 @@ public ValidateApplicantServiceResponse validateApplicantService(
 		 
 		
 		try {
-			
-			 return figservice.validateApplicantService(
-					
-					new ValidateApplicantServiceRequest( 
-							orgPartyId,
-							applPartyId));
-		 
+
+			ValidateOrgApplicantServiceOrdsResponse _ordsResponse = figservice
+					.validateOrgApplicantServiceOrdsResponse(applPartyId, orgPartyId);
+
+			return new ValidateApplicantServiceResponse(_ordsResponse.getStatusMessage(),
+					Integer.parseInt(_ordsResponse.getStatusCode()), _ordsResponse.getValidationResult()
+
+			);
+
 		} catch (FigaroValidationServiceException ex) {
-			logger.error("Exception caught as ValidateApplicantService : " + ex.getMessage()); 
+			logger.error("Exception caught at ValidateApplicantServiceController : " + ex.getMessage());
```

Comment:
I think we want to capture that an exception was thrown, not that we caught an exception, message should be more like `An exception occurred while ... + ex.getMessage()`

Reply:
Agreed to a point. The stack trace provides all the details if we want to look at that. I'm also not sure if the logging can hold the entirety of the underlying cause object. I usually only report the message and look deeper into the stack trace if I need it.

Reply:
I see the full stack trace in splunk for jrcc, going to have a look at it, I still think we should have a constructor that accept a message and a throwable cause.

Reply:
Let me know what you find. We could update the custom exception to take something like this:
public FigaroValidationException(String message, Throwable cause)